### DISCUSSION TITLE
Request for Expedited Review of GitHub Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by Harsimrat0000** [community/community#145781] [GitHub Education]
### Hi everyone,

Hello

I recently applied for the GitHub Student Developer Pack, but I noticed that the review process might take up to 10 days. I am a student currently working on an assignment for my course that requires urgent access to some tools included in the pack.

Would it be possible to expedite the verification of my application? I have already provided the necessary documentation (my student ID). If you need any additional information to speed up the process, I would be happy to provide it promptly.

Thank you for your time and assistance. I greatly appreciate your support in helping students like me access these valuable resources.

**Comment by davevad93** [comment]
Hi @Harsimrat0000 ,

GitHub no longer provides manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Harsimrat0000 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
